lib/rspec/core/runner.rb

```diff
@@ -69,13 +69,13 @@ def self.run(args, err=$stderr, out=$stdout)
           require 'rspec/core/drb'
           begin
             DRbRunner.new(options).run(err, out)
+            return
```

Comment:
Because we don't want to perform the `new(options).run(err, out)` run below if `DRbRunner.new(options).run(err, out)` succeeds.  Previously, that was handled by `new(options).run(err, out)` being in an `else` block and in the `rescue` block, but we can't put it in the `rescue` block anymore.